Test.Helpers/Integration/ProtagonistAppFactoryX.cs

```diff
@@ -0,0 +1,30 @@
+using System.Net.Http;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Test.Helpers.Integration.Infrastructure;
+
+namespace Test.Helpers.Integration;
+
+public static class ProtagonistAppFactoryX
+{
+    public static HttpClient ConfigureIntegrationTestClient<T>(
```

Comment:
I don't know what it is but I think this should have an alternative name as it won't be usable by all services, e.g. Orchestrator won't use as that doesn't have Authentication.

Is the expectation that this'll be the default setup only and if any customisations are required then manually call `.WithConnectionString().WithTestServices()` etc? Thinking of cases where specific tests may need to call `.WithLocalStack()` or add other test services.

Reply:
I'll go with the clunky `ConfigureBasicAuthedIntegrationTestHttpClient` for now